Cabal/doc/installing-packages.rst

```diff
@@ -600,6 +600,15 @@ package:
     ``$version``, ``$compiler``, ``$os``, ``$arch``, ``$abi``,
     ``$abitag``
 
+.. option:: --hidir=dir
+
+    Interfaces of libraries are installed here.
```

Comment:
I'd expand this section a little bit to explain that normally `hidir` is the same as `libdir <\> libsubdir`, except on some broken systems where we must set `libdir` to `""`.